app/views/state_file/questions/post_data_transfer/edit.html.erb

```diff
@@ -2,7 +2,15 @@
 <% content_for :page_title, title %>
 
 <% content_for :card do %>
+  <div class='return-status'>
+    <p>
+      <%= image_tag("icons/check.svg", alt: "") %>
```

Comment:
[dust] i wonder if we need to include this `alt` option if we're not passing anything into it